python/lsst/drp/tasks/reprocess_visit_image.py

```diff
@@ -559,6 +575,23 @@ def run(
 
         return result
 
+    def _copyMaskPlanes(self, exposure, mask):
+        """Copy mask planes from an input Mask to the final Exposure.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            Calibrated exposure; will be modified in place.
+        mask : `lsst.afw.image.Mask`
+            Mask to copy from.
+        """
+        for mp in self.config.copyMaskPlanes:
+            if mp in mask.getMaskPlaneDict().keys():
+                if mp not in exposure.mask.getMaskPlaneDict().keys():
+                    exposure.mask.addMaskPlane(mp)
+                bitMask = mask.getPlaneBitMask(mp)
+                exposure.mask.array |= ((mask.array & bitMask) > 0) * bitMask
```

Comment:
You can transfer all of the mask planes at once:
```
bitMask = mask.getPlaneBitMask(self.config.copyMaskPlanes)
exposure.mask.array |= mask.array & bitMask
```
I don't think you need the loop to add mask planes, either; just reading the new mask will add them to the global plane dict if they weren't there already.